src/.buildkite/publish/docker-compose.yml

```
@@ -8,6 +8,7 @@ services:
     command: /app/src/.buildkite/publish/run.sh
     environment:
       - TEST_POSTGRES_URI=postgres://prisma:prisma@postgres:5432/tests
+      - TEST_POSTGRES_ISOLATED_URI=postgres://prisma:prisma@postgres_isolated:5435/tests
```

```suggestion
      - TEST_POSTGRES_ISOLATED_URI=postgres://prisma:prisma@postgres_isolated:5432/tests
```

That should fix this pipeline